static/src/javascripts/bootstraps/standalone.commercial.ts

```diff
@@ -46,9 +46,20 @@ import type { Modules } from './types';
 
 const { isDotcomRendering, page } = window.guardian.config;
 
-const assetsPath = page.frontendAssetsFullURL ?? page.assetsPath;
+const decideAssetsPath = () => {
+	const assetsPathPort: number | undefined = process.env.OVERRIDE_BUNDLE_PORT
+		? Number(process.env.OVERRIDE_BUNDLE_PORT)
```

Comment:
Why do we convert this to a number?

Reply:
Reading this back, there's probably little point actually. My thinking was that a HTTP port is a number, but if we're just appending it to a string this might be unnecessary...